src/main/java/uk/gov/hmcts/divorce/solicitor/event/SolicitorUpdateApplicant1ContactDetails.java

```diff
@@ -35,7 +35,7 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
     private PageBuilder addEventConfig(final ConfigBuilder<CaseData, State, UserRole> configBuilder) {
 
         return new PageBuilder(configBuilder
-            .event(APPLICANT_UPDATE_CONTACT_DETAILS)
+            .event(APPLICANT_UPDATE_APPLICANT1_CONTACT_DETAILS)
             .forAllStates()
             .name("Update applicant contact info")
```

Comment:
should this also be updated ? `Update applicant 1 contact info`

Reply:
no, in this context it's the applicant or the respondent... don't ask me why